src/Modal/Modal.scss

```diff
@@ -109,3 +109,123 @@
 .modal-footer {
   flex: 0 0 auto;
 }
+
+.pgn__modal-popup__tooltip {
+  border: $border-width solid rgba($black, 20%);
```

Comment:
<img width="486" alt="image" src="https://user-images.githubusercontent.com/2828721/174296374-3f92b504-7cbd-4648-8961-8dcfce6075b4.png">

It seems the border style here is adding a rectangle around the modal popup, overriding the styling potentially passes in by the consumer for the children of `ModalPopup`.

Since the current usage of `ModalPopup` has the consumer supplying `<div className="bg-white p-3 rounded shadow">`, I think it would be appropriate for the consumer to allow provide `border` to add the border to the element with the rounded corners.

The tradeoff is that we won't enforce having a border but we already don't enforce the styles added to the children within `ModalPopup` already.

Reply:
Good, corrected